crates/dojo-lang/src/plugin.rs

```diff
@@ -325,6 +370,19 @@ impl MacroPlugin for BuiltinDojoPlugin {
         item_ast: ast::ModuleItem,
         _metadata: &MacroPluginMetadata<'_>,
     ) -> PluginResult {
+        let package_id = match get_package_id(db) {
+            Option::Some(x) => x,
+            Option::None => return PluginResult {
+                code: Option::None,
+                diagnostics: vec![PluginDiagnostic {
+                    stable_ptr: item_ast.stable_ptr().0,
+                    message: "Unable to find the package ID. Be sure to have a 'package:name' field in your Scarb.toml file.".into(),
```

Comment:
```suggestion
                    message: "Unable to find the package ID. Be sure to have a 'package.name' field in your Scarb.toml file.".into(),
```
In `toml` it's a dot to determine a key into a table. 👍